src/main/kotlin/com/manimdsl/Compiler.kt

```diff
@@ -18,6 +18,12 @@ private fun compile(filename: String, outputVideoFile:String, generatePython: Bo
         exitProcess(1)
     }
 
+    if (stylesheetPath != null && !File(stylesheetPath).isFile) {
```

Comment:
Could we also check if it's JSON? `File(stylesheetPath).extension.toLowerCase() != "json"`?

Reply:
It doesn't have to be called .json actually, but if it's not in a json format the stylesheet handles it and throws an invalid JSON error